src/transforms.jl

```diff
@@ -40,15 +40,42 @@ clrinv(x) = Composition(𝓒(exp.(x)))
 
 Isometric log-ratio transformation of composition `c`.
 """
-function ilr(c::Composition)
-  # TODO
+function ilr(c::Composition{D,SYMS}) where {D,SYMS}
+  log_parts = log.(c.parts .+ eps())
```

Comment:
I think it makes sense to add `eps` right? It is been a while since I touched the other transformations. Maybe we need to do something similar in a future PR.

Reply:
I think it makes sense, though R `compositions` and `ofgulban/compoda` don't seem to implement this. 
Both the CoDa.jl's alr and clr current implementations fail for compositions with zero components. I imagine that working at transformed space for applications like spatial interpolation of compositions, users are bound to encounter this kind of problem/error.
If the eps strategy is adopted, do you think it would be better to implement it at the `Composition` creation methods (maybe as a optional setting, true by default)?

Reply:
Compositions with zero components break dot products as well.

Reply:
That is a good question. Should we maybe consider this `eps` in a separate PR and discussion? Brainstorm the best way forward to handle this corner case? The approach of adding `eps` in the constructor seems interesting as well.